Dockerfile

```diff
@@ -265,7 +262,11 @@ ENV _APP_SERVER=swoole \
     _APP_MAINTENANCE_RETENTION_ABUSE=86400 \
     _APP_MAINTENANCE_INTERVAL=86400 \
     _APP_LOGGING_PROVIDER= \
-    _APP_LOGGING_CONFIG=
+    OPR_EXECUTOR_CONNECTION_STORAGE=file://localhost \
+    OPR_EXECUTOR_INACTIVE_TRESHOLD=600 \
+    OPR_EXECUTOR_NETWORK=openruntimes-runtimes \
+    OPR_EXECUTOR_DOCKER_HUB_USERNAME= \
+    OPR_EXECUTOR_DOCKER_HUB_PASSWORD=
```

Comment:
These variables are not required to build the Appwrite container

Reply:
Removed.